test/validation/api/classification/odp_classification_testsuites.h

```diff
@@ -18,6 +18,7 @@ typedef struct cls_packet_info {
 	bool	vlan_qinq;
 	odp_atomic_u32_t *seq;
 	bool	udp;
+	bool	sctp;
```

Comment:
The existing boolean field udp in cls_packet_info_t was for selecting between UDP and TCP. Now that more protocols are being added, the correct thing to do is to change the boolean to a multi-valued field (like protocol number, uint8_t) instead of keeping adding more and more boolean fields for the same thing.

Reply:
@JannePeltonen  Since this proposal require changes in the validation functions of other PMR terms also, Can i make this change as a different patch?

Reply:
Sure.

Reply:
Ack, added a different patch as part of this series.

Reply:
It would have been better to have the new patch ordered before the validation patches so that the validation patches would not add code that anyway gets changed by the cleanup patch. This ordering makes the total amount of changes slightly bigger (more work to review etc).

Reply:
Ack, sent v4